### DISCUSSION TITLE
Update of Netbox Sync

### DISCUSSION THREAD

**Discussion by Madders04** [bb-Ricardo/netbox-sync#399] [Q&A]
Hello all,

I'm very new in the Netbox area and updated the system to version 4. After that update Netbox sync dont worked anymore. My Question is now: How to update Nebox Sync to the newest version, which supports Netbox version 4.0 and newer.

**Comment by bb-Ricardo** [comment]
You just need to update the sources. If you used git to get the code, a `git pull` inside the project directory should do the update.

**Reply by Madders04** [reply]
Many thanks for your answers. Could you let me know some more details if possible. I'm not that familiar with Git and could you let me know what "sources" you mean.

Many thanks in andvance

**Reply by bb-Ricardo** [reply]
Just wondering, how did you install netbox-sync?

**Reply by Madders04** [reply]
I used the manual in the readme.md. Beginning with git clone. If this helps you?
